.github/workflows/android-continuous.yml

```diff
@@ -1,11 +1,13 @@
 name: Android
 
 on:
+  workflow_dispatch:
```

Comment:
Are those workflow changes only there while you're working on the PR?

Reply:
So I can trigger workflow since I'm not merging into main